controls/update_def.cf.in

```diff
@@ -79,7 +79,8 @@ bundle common update_def
       "input_name_patterns"
         slist => { ".*\.cf",".*\.dat",".*\.txt", ".*\.conf", ".*\.mustache",
                    ".*\.sh", ".*\.pl", ".*\.py", ".*\.rb", ".*\.sed", ".*\.awk",
-                   "cf_promises_release_id", ".*\.json", ".*\.yaml" },
+                   "cf_promises_release_id", ".*\.json", ".*\.yaml", ".*\.csv",
```

Comment:
maybe don't include .csv in 3.12.x, TODO find source of that bit.

Reply:
https://github.com/cfengine/masterfiles/commit/dde9b9ec76820a12800575c919f00934b51b7baa

Reply:
https://tracker.mender.io/browse/CFE-3425
Indicates its in 3.17 and 3.15, so yeah, I wouldn't bring the .csv back to 3.12